Makefile

```diff
@@ -179,15 +179,15 @@ kind-cluster: ## Create a new kind cluster designed for testing with Tilt
 
 .PHONY: manager-core
 manager-core: ## Build core manager binary
```

Comment:
Can you please make the same adjustments in test/infrastructure/docker/{Dockerfile,Makefile}?

Reply:
P.S. I'll fix the golangci-lint issue via https://github.com/kubernetes-sigs/cluster-api/pull/5660

Reply:
let me rebase